### PR TITLE
✨ Landing page

### PR DESCRIPTION
**Pull request by benfoxall**
This focusses the landing page with minimal information needed to get involved with the hack. (moving more detailed info to a second page).

I've added some copy too - I'd very much appreciate any feedback/suggestions for improving it.

image

### Whitespace & assets

I'm using [Prettier](https://prettier.io/) now – so any file I've touched is 🛁 .  (If only GitHub diffs were better!)

I've also moved icons & css into an `./assets` directory - so pages on the root are a bit easier to see.

### local Code of Conduct

We were linking to [Berlin Code of Conduct](https://berlincodeofconduct.org/), but our contact details weren't on it. So I've forked a local version.

### 🌞 Light mode only

The site was looking a bit weird in dark mode, so I've locked it to light for now. Would love to get input from @iainfoxall at some point for this.

### /join notice

Before, "you are agreeing to our code of conduct" sounded a bit like a terms & conditions notice.  I've added a page that makes it more clear what we're asking:

image

When you click the "yes, I can do that":

image

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+All conferences and events should be friendly, safe and inclusive spaces for anyone to attend, and Remote Hack is no different. All organisers and attendees must abide by the [Berlin Code of Conduct](https://berlincodeofconduct.org).
+
+## The Podcast
+
+We'll be recording interviews with the participants throughout the day. Follow along if you like (the episodes are super short...around 2 minutes each).
+
+NOTE: being interviewed is NOT necessary to participate in the hackday.
+
+come follow along at [the RSS feed](https://remotehack.space/live/feed), or just search for "Airquotes the Podcast Airquotes" in your podcatcher.
```
Comment (lpmi-13) [inline about.md:46]: while we're here, I had totally forgot that Spotify is basically the only place you can find the Podcast at the moment...could we update "your podcatcher" to "spotify"?

File: code-of-conduct.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+## Open [Source/Culture/Tech] Citizenship
+
+A supplemental goal of this Code of Conduct is to increase open [source/culture/tech] citizenship by encouraging participants to recognize and strengthen the relationships between our actions and their effects on our community.
+
+Communities mirror the societies in which they exist and positive action is essential to counteract the many forms of inequality and abuses of power that exist in society.
+
+If you see someone who is making an extra effort to ensure our community is welcoming, friendly, and encourages all participants to contribute to the fullest extent, we want to know.
+
+## Expected Behavior
```
Comment (lpmi-13) [inline code-of-conduct.md:30]: ah HA! A secret American! Can we "u" this up, please?

File: code-of-conduct.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+Harassment includes: harmful or prejudicial verbal or written comments related to gender, sexual orientation, race, religion, disability; inappropriate use of nudity and/or sexual images (including presentation slides); inappropriate depictions of violence (including presentation slides); deliberate intimidation, stalking or following; harassing photography or recording; sustained disruption of talks or other events; inappropriate physical contact, and unwelcome sexual attention.
+
+## Consequences of Unacceptable Behavior
+
+Unacceptable behavior from any community member, including sponsors and those with decision-making authority, will not be tolerated.
+Anyone asked to stop unacceptable behavior is expected to comply immediately.
+
+If a community member engages in unacceptable behavior, the community organizers may take any action they deem appropriate, up to and including a temporary ban or permanent expulsion from the community without warning (and without refund in the case of a paid event).
+
+## If You Witness or Are Subject to Unacceptable Behavior
```
Comment (lpmi-13) [inline code-of-conduct.md:51]: ...nvmd...we got US spelling all over this thing...I guess just leave as is.

**Comment by spikeheap** [comment]
❤️ to the signup workflow. That’s so much more explicit, clear and friendly. 

Could we make that “Yes I can do that” disclosure more obviously a click-site? Maybe bigger text and/or some highlighting of the region would make it more obvious. Happy to merge this and follow that up separately, ofc.

I like the minimal approach 👍.

File: about.md (hunk 1 above)
Comment (benfoxall) [reply about.md:46]: Ah, totally.

Do you know, perhaps we should move the podcast to a page of its own?  It's a really nice way of finding out about the kind of stuff we're working on.

Comment (lpmi-13) [reply about.md:46]: Podcast on its own page totally works!

File: code-of-conduct.md (hunk 2 above)
Comment (benfoxall) [reply code-of-conduct.md:30]: "u" got it :+1:

File: code-of-conduct.md (hunk 3 above)
Comment (benfoxall) [reply code-of-conduct.md:51]: It's cool, I know how to find+replace 💅 

keyboard

File: about.md (hunk 1 above)
Comment (benfoxall) [reply about.md]: # 🙌 

[image](https://deploy-preview-45--wizardly-wiles-9029ce.netlify.app/podcast)

**Pull request merged**